src/types/types.ts

```diff
@@ -294,3 +294,163 @@ export interface TimeResult {
     travelTime: number
   }[]
 }
+
+/**
+ * A store (also called 'depot' sometimes) to which the respective vehicles/transports and orders are associated.
+ * Each store will be optimized individually and independently.
+ * @param {string} uuid Unique ID that is required to be set so that Order and Vehicle can reference to their respective store.
+ * @param {string} name Name of the store - cannot be longer than 256 chars.
+ * @param {Address} address Location of the store. For store addresses the geocoordinates(lat, lng) must exist already.
+ */
+export interface Store {
+  uuid: string;
```

Comment:
is it required to be a UUID? or can it be any arbitrary unique id? 

if second, please rename to `id`

Reply:
this question applies to all occurrences of uuid as param...

Reply:
Didnt see you had commented on this months ago. But yes, this is necessary because the fleetplanner service returns objects which use these parameter names (https://docs.targomo.com/fleetplanner/). And the resulting object from the service is not changed before returning it in the FleetsClient.fetch method. So it has to be called uuid so that the typings are correct of the object that we return.